WordPress/src/main/java/org/wordpress/android/ui/people/PeopleListFragment.java

```diff
@@ -0,0 +1,107 @@
+package org.wordpress.android.ui.people;
+
+import android.app.Activity;
+import android.app.ListFragment;
+import android.content.Context;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.AdapterView.OnItemClickListener;
+
+import org.wordpress.android.R;
+import org.wordpress.android.datasets.PeopleTable;
+import org.wordpress.android.models.Person;
+
+import java.util.List;
+
+public class PeopleListFragment extends ListFragment implements OnItemClickListener {
+    private static String ARG_LOCAL_TABLE_BLOG_ID = "local_table_blog_id";
+
+    private int mLocalTableBlogID;
+    private OnPersonSelectedListener mListener;
+
+    public static PeopleListFragment newInstance(int localTableBlogID) {
+        PeopleListFragment peopleListFragment = new PeopleListFragment();
+        Bundle bundle = new Bundle();
+        bundle.putInt(ARG_LOCAL_TABLE_BLOG_ID, localTableBlogID);
+        peopleListFragment.setArguments(bundle);
+        return peopleListFragment;
+    }
+
+    @Override
+    public void onAttach(Context context) {
```

Comment:
These two `onAttach` implementations only exist in order to set the `OnPersonSelectedListener`. It might be simpler to have a `setOnPersonSelectedListener` method instead.